packages/esm-care-panel-app/src/care-panel/care-panel.component.tsx

```diff
@@ -46,44 +47,43 @@ const CarePanel: React.FC<CarePanelProps> = ({ patientUuid, formEntrySub, launch
   if (error) {
     return <ErrorState error={error} headerTitle={t('carePanelError', 'Care panel')} />;
   }
-
-  if (Object.keys(enrollments).length === 0) {
-    return (
-      <div className={styles.careProgramContainer}>
-        <CarePrograms patientUuid={patientUuid} />
-      </div>
-    );
-  }
+  console.info(programEnrolled);
```

Comment:
can we remove this

Reply:
@jabahum This has been removed in the subsequent commit